### PR TITLE
DP 1347 rmv wait time component update

### PR DESCRIPTION
**Pull request by jesconstantine**
This PR alters the markup, presentation, and variable structure for the wait time component, based on the screenshot from visual design work done in [DP-822](https://jira.state.ma.us/browse/DP-822).

User Story: https://jira.state.ma.us/browse/DP-1347

Looking Design and Front End UAT.

## To Test

Either use screenshots below or:

1. Pull down branch and build styleguide
2. See molecule isolated at: /?p=molecules-wait-time
3. See component on rmv location page at: /?p=pages-LOC-Southbridge-RMV

### From DP-822
1-17_velir_mass_gov_waittimecomponent_psd___100___icon_copy_2__rgb_8__

#### Small
image

#### Full
image

### From work done (No wait time)
image

### From work done (closed)
image

### From work Done (with error state)
image

### CODE REVIEW

**Comment by jesconstantine** [comment]
@legostud I'm not positive that this PR should be merged, we mentioned removing this component all together.  But I thought it was a good place to start since you got my 85% of the way there with the existing component.

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss [hunk 1]
```diff
@@ -36,23 +36,18 @@
     display: flex;
       flex-wrap: wrap;
       align-items: stretch;
+    margin-bottom: 1rem;
```
Comment (gender404) [inline styleguide/source/assets/scss/02-molecules/_wait-time.scss:36]: The display: flex and it's two other attributes can be removed

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss [hunk 2]
```diff
@@ -36,23 +36,18 @@
...
       flex-wrap: wrap;
       align-items: stretch;
+    margin-bottom: 1rem;
   }
 
   &__item {
     flex-basis: auto;
     flex-grow: 0;
     flex-shrink: 0;
-    margin-bottom: 20px;
```
Comment (gender404) [inline styleguide/source/assets/scss/02-molecules/_wait-time.scss:45]: flex-... aren't needed

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss [hunk 3]
```diff
@@ -36,23 +36,18 @@
...
-    width: 50%;
-
-    &:nth-child(even) {
-      border-left: 1px solid;
-      padding-left: 38px;
-    }
+    width: 100%;
+  }
```
Comment (gender404) [inline styleguide/source/assets/scss/02-molecules/_wait-time.scss]: with flex removed, width: 100% isn't needed

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss [hunk 4]
```diff
@@ -63,11 +58,14 @@
     text-transform: uppercase;
   }
 
-  &__time {
+  &__time
+  {
     font-size: 2.125rem;
```
Comment (gender404) [inline styleguide/source/assets/scss/02-molecules/_wait-time.scss:52]: should be &__time {

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss [hunk 5]
```diff
@@ -63,11 +58,14 @@
...
+  {
     font-size: 2.125rem;
   }
 
-  &__units {
-    font-size: 1.5rem;
+  &__refreshed
+  {
+    font-size: 1.125rem;
+    font-style: italic;
```
Comment (gender404) [inline styleguide/source/assets/scss/02-molecules/_wait-time.scss]: font-size: italic; should be in the 06-themes version of this file.

**Review by gender404 — CHANGES_REQUESTED** [review]

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss (hunk 1 above)
Comment (jesconstantine) [reply styleguide/source/assets/scss/02-molecules/_wait-time.scss:36]: Thanks for all of your feedback @legostud - addressed in https://github.com/massgov/mayflower/pull/301/commits/6f6db2c73f216c0c2424204d43f73ffc661d9588

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss (hunk 2 above)
Comment (jesconstantine) [reply styleguide/source/assets/scss/02-molecules/_wait-time.scss:45]: addressed in https://github.com/massgov/mayflower/pull/301/commits/6f6db2c73f216c0c2424204d43f73ffc661d9588

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss (hunk 3 above)
Comment (jesconstantine) [reply styleguide/source/assets/scss/02-molecules/_wait-time.scss]: addressed in https://github.com/massgov/mayflower/pull/301/commits/6f6db2c73f216c0c2424204d43f73ffc661d9588

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss (hunk 4 above)
Comment (jesconstantine) [reply styleguide/source/assets/scss/02-molecules/_wait-time.scss:52]: addressed in https://github.com/massgov/mayflower/pull/301/commits/6f6db2c73f216c0c2424204d43f73ffc661d9588

File: styleguide/source/assets/scss/02-molecules/_wait-time.scss (hunk 5 above)
Comment (jesconstantine) [reply styleguide/source/assets/scss/02-molecules/_wait-time.scss]: addressed in https://github.com/massgov/mayflower/pull/301/commits/6f6db2c73f216c0c2424204d43f73ffc661d9588

**Review by gender404 — APPROVED** [review]

**Comment by gender404** [comment]
I'm ok with merging this in.

**Comment by gender404** [comment]
If we decide to merge this in, could this please be added to the release notes for Sprint 11.

https://github.com/massgov/mayflower/blob/dev/release-notes.md

thanks.

**Comment by jesconstantine** [comment]
Thanks again, @legostud ... just waiting for design input for this component in general.  I'll label WIP to make sure it doesn't get merged in until that happens.

**Comment by gender404** [comment]
@reenybeeny looked at on my computer and didn't have any immediate issues with it.  She usually reviews it after it has been merged into the dev environment.

**Comment by jesconstantine** [comment]
Thanks @legostud and @reenybeeny -- I'll talk to @isaacchansky to get his thoughts on adding / removing this component from Mayflower.  If we merge, I'll certainly also add this to the release notes.

**Comment by isaacchansky** [comment]
This looks good to me

**Review by isaacchansky — APPROVED** [review]

**Pull request closed without merging**